src/std/text.ab

```diff
@@ -1,14 +1,26 @@
-/// Replaces all occurences of a pattern in the content with the provided replace text.
+/// Replaces all occurrences of a pattern in the content with the provided replace text.
+#[allow_absurd_cast]
 pub fun replace(source, search, replace) {
-    return "\$\{source//{search}/{replace}}"
+    search = trust $ echo \$\{{nameof search}//\\\\/\\\\\\\\} $
+    // TODO: Revisit after https://github.com/amber-lang/amber/issues/632
+    const bash_major = trust $ echo \"\$\{BASH_VERSINFO[0]}\" $ as Num
+    const bash_minor = trust $ echo \"\$\{BASH_VERSINFO[1]}\" $ as Num
+    if bash_major > 5 or (bash_major == 5 and bash_minor > 1) {
+        replace = trust $ echo \$\{{nameof replace}//\\\\/\\\\\\\\} $
+    } else {
+        replace = trust $ echo \$\{{nameof replace}//\\\\/\\\\} $
+    }
```

Comment:
I know this is dirty. When https://github.com/amber-lang/amber/issues/632 is resolve, this could be improved.